src/components/dashboard/armFlows/ArmFlowFour.tsx

```diff
@@ -7,26 +7,30 @@ import {
   ResponsiveStepWrapper,
 } from 'components/common'
 import SageForm from 'components/form/SageForm'
-import { FORM_IDS } from 'components/form/types'
+import { FORM_IDS, WHO_CONTROLS_DATA_OPTIONS } from 'components/form/types'
 import { ReactComponent as Globe } from 'assets/consent/globe.svg'
 import { PAGE_ID_FIELD_NAME, PAGE_ID } from 'constants/constants'
 
 type ArmFlowFourProps = {
   step: number
-  maxSteps: number
-  handleNext: (pageId?: string) => void
+  handleNext: (fields?: object) => void
   handleBack: () => void
-  handleComplete: (pageId?: string) => void
+  handleComplete: (fields?: object) => void
   updateClientData: Function
+  RankedChoice: any
+  clientData: any
 }
 
+const maxSteps = 10
```

Comment:
Are we using this variable? 
If we are, we already have a getArmFlowMaxSteps function, there's no need for duplication.

Reply:
@jwilliam706 done.